pingouin/pairwise.py

```diff
@@ -121,7 +121,8 @@ def pairwise_tests(
 
         .. versionadded:: 0.2.9
     return_desc : boolean
-        If True, append group means and std to the output dataframe
+        If True, append group means and std for parametrics=True (default) to the output dataframe, if
```

Comment:
please replace with "If True, append group means and std to the output dataframe when ``parametric=True`` (defaut), or median and IQR when ``parametric=False``."